docs/about.md

```diff
@@ -1,3 +1,3 @@
-# dcat-4C-ap
+# ChemDCAT-AP
```

Comment:
Should this be also with a hyphen: Chem-DCAT-AP ? That would match the naming in code. (same for first headline in docs/index.md)